src/migrations/hotbar-store/5.0.0-alpha.0.ts

```diff
@@ -22,7 +22,11 @@ export default migration({
       }
 
       hotbar.items.push({
-        entity: { uid: cluster.id },
+        entity: {
```

Comment:
We need to create a new migration because this will not be applied to existing beta users.

Reply:
Created new migration in `src/migrations/hotbar-store/5.0.0-beta.4.ts`